build: `ng_package` no longer generate minified UMDs

alan-agius4 opened the pull request:
In version 12, applications will only be allowed to be built in Ivy, this makes the **minified** UMDs redundant since they cannot be processed by NGCC.

With this change, we remove the minified UMDs from the generated APF package.

petebacondarwin commented on packages/bazel/src/ng_package/ng_package.bzl:
```diff
@@ -30,14 +30,7 @@ def _debug(vars, *args):
 
 _DEFAULT_NG_PACKAGER = "//@angular/bazel/bin:packager"
 _DEFAULT_ROLLUP_CONFIG_TMPL = "//:node_modules/@angular/bazel/src/ng_package/rollup.config.js"
-_DEFALUT_TERSER_CONFIG_FILE = "//:node_modules/@angular/bazel/src/ng_package/terser_config.default.json"
```
DEFALUT 🤣

petebacondarwin commented on packages/bazel/src/ng_package/ng_package.bzl:
```diff
@@ -136,46 +129,6 @@ WELL_KNOWN_GLOBALS = {p: _global_name(p) for p in [
 # TODO(gregmagolan): clean this up
 _DEPSET_TYPE = "depset"
 
-def _terser(ctx, input, output):
```
YAY!

petebacondarwin reviewed (approved):
Bazel is not my area of expertise but this LGTM.
Given that this is a breaking change, do we need to highlight that in the commit?
Also should the APF spec document be updated to remove the minified UMD file references?

alan-agius4 commented:
> Also should the APF spec document be updated to remove the minified UMD file references?

Yeah, I'll do that once this lands. Since the APF doc is a Google doc

> Given that this is a breaking change, do we need to highlight that in the commit?
SGTM

josephperrott reviewed (approved):
LGTM

One nit, the commit message breaking change message says


> Minified UMD bundles are _**not**_ longer included in the distributed NPM packages.


I believe it should say


> Minified UMD bundles are _**no**_ longer included in the distributed NPM packages.

alan-agius4 commented:
@josephperrott updated the commit message.

alan-agius4 committed:
build: `ng_package` no longer generate minified UMDs

In version 12, applications will only be allowed to be built in Ivy, this makes the minified UMDs redundant since they cannot be processed by NGCC.

With this change, we remove the minified UMDs from the generated APF package.

BREAKING CHANGE:  Minified UMD bundles are no longer included in the distributed NPM packages.
